research/README.md

```diff
@@ -44,3 +44,18 @@ certain TensorFlow versions or Python versions.
 
 [[4] A. Pakman, Y. Wang, C. Mitelut, J. Lee, L. Paninski. "Neural Clustering
 Processes." ICML 2020](https://arxiv.org/abs/1901.00409)
+
+`third_party` directory contains research project contributions from the open source
+community. 
```

Comment:
I don't think that's the intent of the third_party directory here. @DualityGap, any thoughts on this? My understanding is that this indicates that contributions here are built on top of other third_party implementations (https://github.com/microsoft/denoised-smoothing in this example). So, I'd suggest removing these 2 lines. As another counterexample, the gnn-survey directory was also a community contribution but that was a from-scratch implementation.

Reply:
I see. In this case, you are right. I have reused some of the bits from the official implementation (with all the citations) and reimplemented bits that were absolutely necessary to make things work in TensorFlow and Keras. I will reword the phrasing then.